src/client/QXmppCallStream.h

```diff
@@ -34,6 +34,8 @@ class QXmppCallStreamPrivate;
 class QXmppIceConnection;
 class QXmppCall;
 class QXmppCallPrivate;
+template<typename _Signature>
+    class function;
```

Comment:
STL classes shouldn't be forward-declared generally (as in, the standard explicitly forbids this), and this decl is a no-op anyway since it's not in `std::`.

Reply:
Thanks, its fixed now. Apparently the Qt 5.15 headers are leaking the functional header themselves, so it still compiled for me.